### ISSUE TITLE
suggestion: minimal working example in crate's docs/Readme

### ISSUE THREAD

**Issue by lspitzner** [alexcrichton/futures-rs#299]
readme currently contains no example; the tutorial starts by using `tokio` stuff (making it far from minimal; some might even think that you need other crates to make use of futures); and the crate's doc only contain some non-complete (and thus non-working) code.

I think the code on `futures::sync::oneshot::channel` would be a nice, minimal working example. Although I'd perhaps fork the sender, not the receiver, but that does not matter too much.

Thanks for providing the crate!

**Comment by alexcrichton** [comment]
I think we're not covered with https://tokio.rs, so I'm going to close in favor of that.

**Comment by lspitzner** [comment]
Could you rephrase that? I don't understand.

**Comment by alexcrichton** [comment]
Oops "now covered" is what I meant.

In general https://tokio.rs is a good entry point for documentation and examples for futures.
